test/rule_test.dart

```diff
@@ -319,19 +319,16 @@ AnnotationMatcher matchesAnnotation(
         String message, ErrorType type, int lineNumber) =>
     AnnotationMatcher(Annotation(message, type, lineNumber));
 
-testEach(Iterable<Object> values, bool f(String s), Matcher m) {
-  values.forEach((s) => test('"$s"', () => expect(f(s), m)));
-}
-
-testEachInt(Iterable<Object> values, bool f(int s), Matcher m) {
+testEach<T>(Iterable<T> values, bool f(T s), Matcher m) {
```

Comment:
Could we just import the definition from `test_utils.dart` rather than duplicating it here?